src/EnergyPlus/WeatherManager.cc

```diff
@@ -10560,7 +9192,7 @@ namespace WeatherManager {
         bool epwFileExists;
         bool epwHasLeapYear(false);
 
-        if (!OADryBulbAverage.OADryBulbWeatherDataProcessed) {
+        if (!this->OADryBulbWeatherDataProcessed) {
```

Comment:
This is in a method on the this struct, but there's only ever one of them and it always acts on it. I just made it use the `this` pointer instead of using the only instance of this struct directly.

Reply:
OK, this seems fine, I think.  Basically there will only ever be one instance of a AnnualMonthlyDryBulbWeatherData.  And you are calling this method from somewhere and you are just altering the value of the single instance.  But it really doesn't matter, I mean if you had more than one for whatever reason, this would still be the right thing to do.  Right?

Reply:
Yes, this would be the right thing to do in that case. Previously it was kind of weird because this method was on the struct/class but instead of operating on the struct it was attached to, operated on the one and only `OADryBulbAverage`. If there ever were more than one, it wouldn't work right.